### DISCUSSION TITLE
Certification for CSA CCM

### DISCUSSION THREAD

**Discussion by corydorning** [Checkmarx/kics#5376] [Ideas] [answered]
Would be nice if KICS queries could be easily configured for other standards. Currently it is certified for CIS AWS Benchmarks 1 & 2, but integration with the Cloud Security Alliance's (CSA) Cloud Controls Matrix (CCM) would be be really beneficial. Especially if trying to appeal to corporate adoption.

This is currently one such gap in getting company adoption for us.

https://cloudsecurityalliance.org/research/cloud-controls-matrix/

**Comment by kaplanlior** [comment, marked as answer]
Thanks @corydorning for raising this, marked as a feature request.

I'm curious if this is an issue for other organizations (please do comment if that's the case).
